app/i18n/de/feedback.php

```diff
@@ -35,7 +35,7 @@
 	),
 	'conf' => array(
 		'error' => 'Während der Speicherung der Konfiguration trat ein Fehler auf',
-		'query_created' => 'Abfrage "%s" ist erstellt worden.',
+		'query_created' => 'Abfrage “%s” ist erstellt worden.',
```

Comment:
this quotation marks are not the German style, it is English style.
German quotation marks are on the bottom line (starting quotation mark) and and the top line (closing quotation mark).

If you want to learn more: https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen (sorry, it is only available in German)

Reply:
All German files fixed accordingly